metadata/scripts/platemap_vis.r

```diff
@@ -78,7 +82,10 @@ plates_partial_gg <-
     theme(plot.title = element_text(size = 15, face = "bold")) +
     ggplot2::geom_point(aes(shape = platemap_df_partial$cell_type)) +
     ggplot2::scale_shape_discrete(name = "Cell Type") +
-    ggplot2::scale_fill_discrete(name = "Treatment")
+    ggplot2::scale_fill_manual(name = "Treatment",
+                               values = c("compound" = "#F8766D", 
+                                          "DMSO" = "#00BFC4",
+                                          "TGFRi" = "#7CAE00")) # Assign the default colors to match the previous full platemap layouts
```

Comment:
If you find yourself resusing these colour schemes and themes consider using and importing a themes file:
https://github.com/WayScience/phenotypic_profiling/blob/main/7.figures/figure_themes.r

Reply:
This will definitely be on my to-do list once we begin figure generation for a future manuscript. Thanks for bringing this up!